corehq/sql_db/connections.py

```diff
@@ -100,20 +108,31 @@ def get_engine(self, engine_id=DEFAULT_ENGINE_ID):
         """
         return self._get_or_create_helper(engine_id).engine
 
-    def get_load_balanced_read_db(self, engine_id, default=None):
+    def get_load_balanced_read_engine_id(self, engine_id, default=None):
```

Comment:
why the rename?

Reply:
This is not a rename, there were three functions where we could have one. The diff makes it look like a rename, but it's not. Though I could use the name `get_load_balanced_read_db ` instead of the other